src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -126,9 +128,12 @@ public void Constructor_BoundSolution_SyncsAndStartsMonitoring()
             var issuesProvider = new SonarQubeIssuesProvider(mockSqService.Object, mockTracker.Object, mockTimerFactory.Object);
 
             // Assert - issues are fetched and timer is started
-            VerifyServiceGetIssues(Times.Once());
             VerifyTimerStart(Times.Once());
             timerRunning.Should().Be(true);
+
+            // The issues are fetched on a background thread - short pause to allow it complete
```

Comment:
Not ideal, but it seems to be reliable enough. I'll have a quick look to see if I can remove the delay by using a _EventWaitHandle_ like in one of the later tests.